extension/otto/gohan_core.go

```diff
@@ -141,23 +141,32 @@ func init() {
 		}
 
 		err = env.Load("<Gohan built-ins>", `
-		var gohan_handler = {}
+		var gohan_handler = {};
+		var gohan_caller = "";
 		function gohan_register_handler(event_type, func){
 		  if(_.isUndefined(gohan_handler[event_type])){
 		    gohan_handler[event_type] = [];
 		  }
-		  gohan_handler[event_type].push(func)
+		  var id = gohan_uuid();
+		  gohan_handler[event_type].push({fn:func,id:id})
+		  gohan_log_debug("gohan built-in: register handler: type = '" + event_type.toString() +
```

Comment:
make the log message shorter, "Registering handler: ..." will be just enough IMO